cpt-dl/src/cptdl/catalog.py

```diff
@@ -431,5 +445,15 @@
     'NNRP': {'iri': 'http://iridl.ldeo.columbia.edu/SOURCES/.NOAA/.NCEP-NCAR/.CDAS-1/.MONTHLY/', 'info': 'https://doi.org/10.1175/1520-0477(1996)077%3C0437:TNYRP%3E2.0.CO;2/', 'institution': 'NOAA/NCAR', 'dlauth_required': False, 'limits': {'start': '1949-01-01', 'end': -1}}, 	
 }
 
-
-
+deprecations = {
+    f'SEAS51.{var}': (
+        f'SEAS51b.{var}',
+        'https://iridl.ldeo.columbia.edu/SOURCES/.EU/.Copernicus/.CDS/.C3S/.ECMWF/.dataset_documentation.html'
+    )
+    for var in ('PRCP', 'T2M', 'GPH', 'SST', 'UA', 'UV')
+} | {
+    'SPEAR.SST': (
+        'SPEARb.SST',
+        'https://iridl.ldeo.columbia.edu/SOURCES/.Models/.NMME/.GFDL-SPEAR/.HINDCAST/.MONTHLY/.dataset_documentation.html'
+    ),
+}
```

Comment:
Is it necessary to use deprecation for the old stations in Chile if we will no longer have any station (new or old) in the dictionary in catalog.py because they will be directly in the notebook?"